litestar/contrib/sqlalchemy/dto.py

```diff
@@ -94,7 +94,7 @@ def _(
                 default=default,
                 parsed_type=parsed_type,
                 default_factory=default_factory,
-                dto_field=elem.info.get(DTO_FIELD_META_KEY),
+                dto_field=elem.info.get(DTO_FIELD_META_KEY, DTOField()),
```

Comment:
Why not use Empty to signify this?

Reply:
Because `FieldDefinition.dto_field` is typed `DTOField`, not `DTOField | Empty` or `DTOField | None`. That is intentional to make it easier to use downstream. I would have given the field a default value on the dataclass but without `kw_only` for dataclasses in 3.8, that makes the type impossible to subclass.